src/app/pages/project/project-form/project-form.component.ts

```diff
@@ -45,7 +47,12 @@ export class ProjectFormComponent implements OnInit {
 		this.projectForm = this.formBuilder.group({
 			'name': ['', [Validators.required]],
 			'description': [''],
-			'team': ['', Validators.required]
+			'team': [null, Validators.required]
+		});
+
+		this.teamProjectService.getTeam().subscribe(choosedTeam => {
```

Comment:
aqui temos um problema quando tentarmos acessar a tela na visão de editar/detalhe em vez de new project, pq em vez de carregar o team do project, ele vai carregar seja la qual for o team q ta la no subject do servico.

Um outro possível problema é se caso quisermos acessar essa tela de uma outra forma, que n seja pelos links q tu criou (ex: um btn new project no menu, sei la), nesse caso, o campo team n vai ta vazio, vai ta loaded com o team q ta carregado no serviço